_data/sites.json

```diff
@@ -7386,6 +7386,14 @@
         ]
     },
 
+    {
+        "name": "Ripe NCC",
+        "difficulty": "impossible",
```

Comment:
Can you please provide the url, preferably one that endorses that it's not possible to delete an account?

Reply:
There's no mention of account deletion anywhere to be found, not even the privacy policy.

Reply:
```suggestion
        "name": "Ripe NCC",
        "difficulty": "impossible",
        "url": "",
```
might be feasible.

Reply:
If there's no privacy policy or faq link, I'd rather you just added the plain url to the site. All of our links are clickable, so we require such.

Reply:
Changes pushed, and commits merged.